airontools/constructors/layers.py

```diff
@@ -69,6 +69,9 @@ def layer_constructor(
             images and 4 for 3D images).
             advanced_reg (bool): Whether to automatically set advanced regularization. Useful to quickly make use of all
             the regularization properties.
+            is_activated_custom (bool): Whether to use a custom Dense layer with keras Functional API or not
```

Comment:
```suggestion
```
Why is it relevant to mention whether the custom layer is activated? perhaps it could be assumed to be activated by the fact that the custom layer given by custom_layer argument